benchopt/cli/tests/test_cli.py

```diff
@@ -241,6 +242,21 @@ def test_config_file(self, no_debug_log):
         n-repetitions: {n_reps}
         max-runs: 0
         """
+        self.run_config_file(config, n_reps, no_debug_log)
+        
+    def test_config_file_single_line(self, no_debug_log):
+        n_reps = 2
+        
+        config = f"""
+        objective: test-objective
+        dataset: test-dataset
+        solver: test-solver[param1=42]
+        n-repetitions: {n_reps}
+        max-runs: 0
+        """ 
+        self.run_config_file(config, n_reps, no_debug_log)     
```

Comment:
The `run_config` function is making a lot of tests at once.
As this test is supposed to be a non-regression test for the single line config, I would keep the two test separated, with the `test_config_file_single_line` just checking that it runs (so the first part of the test).